build/dependencies.props

```
@@ -19,7 +19,7 @@
   </PropertyGroup>
 
   <!--
-    Attempt to keep these mostly aligned with https://github.com/dotnet/toolset/blob/master/env/Versions.props
+    Attempt to keep these mostly aligned with https://github.com/dotnet/toolset/blob/main/env/Versions.props
```

```suggestion
    Attempt to keep these mostly aligned with https://github.com/dotnet/toolset/blob/master/eng/Versions.props
```
No 'main' branch in this repo. It still has a 'master' branch and a nearly-empty 'archive' branch for some reason.